src/mach/load_command.rs

```diff
@@ -371,44 +371,137 @@ pub const SIZEOF_DYLINKER_COMMAND: usize = 12;
 // struct XXX_thread_state state   thread state for this flavor
 // ...
 #[repr(C)]
-#[derive(Debug, Clone, Copy, Pread, Pwrite, SizeWith)]
-pub struct ThreadCommand {
+#[derive(Debug, Copy, Clone)]
+pub struct ThreadCommand<'a> {
```

Comment:
I don't like the lifetime here at all.

So I just got done removing it from Section, which was a mistake in the first place, and I'm going to try to remove it from Segment (probably not tho).

The reason is that it makes construction of the load command more difficult (the bytes can't be written as a stream).

Also, the implementation for pwrite will be complicated.

It means that someone has to write the bytes for you, and insert them into the threadcommand (e.g., someone pre-marshalled the bytes in the appropriate endianness, and type (this last part not so relevant for threadcommand, but was for segment/section)).

Then you have to decide who writes the bytes in the struct, either the top reference of the bytes, or the threadcommand pwrite impl.

Nevertheless, if you think it's worthwhile to include the bytes, and don't see a better way (or if its not worth the pain), i'm fine with it.  I don't think I'll be actually ever be writing this load command since it would use the more modern one.